### ISSUE TITLE
Allow ORT built-in OSADL rules to be used

### ISSUE THREAD

**Issue by sschuberth** [eclipse-apoapsis/ort-server#613]
Similar to like in [ORT's Jenkins integration](https://github.com/oss-review-toolkit/ort/tree/main/integrations/jenkins), it should be possible for an evaluator run to use the OSADL rules built into ORT, see

![image](https://github.com/eclipse-apoapsis/ort-server/assets/349154/cebc449d-c04e-4e99-88fe-e07a5ac0c77f)

Note that if this option is enabled, the OSADL rules are used *in addition* to any other rules specified.

**Comment by Etsija** [comment]
I'm unsure if this is UI issue only: I don't see that there is currently any job configuration parameter for enabling OSADL rules in the Evaluator job.

**Comment by mnonnenmacher** [comment]
> I'm unsure if this is UI issue only: I don't see that there is currently any job configuration parameter for enabling OSADL rules in the Evaluator job.

This is correct, currently it is not possible to use a rule set from the classpath, this would have to be implemented first.
